### PR TITLE
Add operate value docs

### PR DESCRIPTION
**Pull request by Zelldon**
* Replaces some variables with already defined global ones.
* Rename properties to be aligned with other names (created -> enabled)
* Document operate values, following the https://github.com/camunda-community-hub/camunda-cloud-helm/blob/main/CONTRIBUTING.md#documentation
* Move operate values to parent values.yaml file. To reduce duplication and avoid drifting away (values). Values are specified and documented in the parent values.yaml file, this will overwrite anyway values from the subchart. This should create one single place of truth we only need to manage.

related to #124

### CODE REVIEW

**Review by npepinpe — APPROVED** [review]
:+1: 

I assume there will be a follow up at some point to document everything in the README?

File: charts/ccsm-helm/values.yaml [hunk 1]
```diff
@@ -232,10 +232,74 @@ zeebe:
...
-
+# Operate configuration for the operate sub chart.
 operate:
+  # Enabled if true, the operate deployment and its related resources are deployed via a helm release
   enabled: true
 
+  # Logging configuration for the operate logging. This template will be directly included in the operate configuration yaml file
+  logging:
+    level:
+      ROOT: INFO
```
[the remark this answers is not in the document]
Comment (Zelldon) [reply charts/ccsm-helm/values.yaml:243]: I totally agree. I was also unsure about these properties. It seems also that they are out dated. I asked for feedback from the operate team. https://camunda.slack.com/archives/C9B5270DA/p1643803696900629

In the deployment guide it seems to reference only the log4j file, so we could do that. https://docs.camunda.io/docs/self-managed/operate-deployment/configuration/#logging

[the remark this answers is not in the document]
Comment (Zelldon) [reply charts/ccsm-helm/values.yaml:243]: I would do that separately then

**Comment by Zelldon** [comment]
> I assume there will be a follow up at some point to document everything in the README?

Exactly this was my plan :)

Thanks for the review!

**Pull request merged**